From: Helmut Grohne <helmut@subdivi.de>
Date: Fri, 5 Oct 2018 16:33:27 +0200
Subject: Don't test for stdio.h in fullpath

Closes: #910322
---
 Makefile | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/Makefile b/Makefile
index 04085c2..33ec176 100644
--- a/Makefile
+++ b/Makefile
@@ -402,17 +402,6 @@ all-test:
 	    echo "      ***"; \
 	    exit 1; \
 	fi
-	@if [ "`ls $(MACOSX_stdio) 2> /dev/null`" != "" ]; then \
-	    : ; \
-	elif ! test -f /usr/include/stdio.h; then \
-	    echo "      ***"; \
-	    echo "      *** Error: /usr/include/stdio.h is not installed!"; \
-	    echo "      ***"; \
-	    echo "      *** Install Software Development (gcc) package"; \
-	    echo "      *** for Ubuntu: sudo apt-get install build-essential"; \
-	    echo "      ***"; \
-	    exit 1; \
-	fi
 	@if ! type gs >/dev/null 2>&1; then \
 	    echo "      ***"; \
 	    echo "      *** Error: gs is not installed!"; \
