From: Quirinux GNU/Linux <49002514+quirinux-so@users.noreply.github.com>
Date: Thu, 2 Sep 2021 14:19:18 +0200
Subject: getweb: use quirinux.org mirror as the original site is down

Origin: https://github.com/koenkooi/foo2zjs/pull/3
Debian-Bug: #992282
---
 getweb.in | 62 +++++++++++++++++++++++++++++++-------------------------------
 1 file changed, 31 insertions(+), 31 deletions(-)

diff --git a/getweb.in b/getweb.in
index 3322de6..1eff282 100755
--- a/getweb.in
+++ b/getweb.in
@@ -176,7 +176,7 @@ getone() {
     case "$1" in
     lj1000|1000)
 	gettgz \
-	    http://foo2zjs.rkkda.com/firmware sihp1000.tar.gz \
+	    https://www.quirinux.org/printers sihp1000.tar.gz \
 	    ""
 	#echo "provided"
 	#getexe \
@@ -188,7 +188,7 @@ getone() {
 	;;
     lj1005|1005)
 	gettgz \
-	    http://foo2zjs.rkkda.com/firmware sihp1005.tar.gz \
+	    https://www.quirinux.org/printers sihp1005.tar.gz \
 	    ""
 	#echo "provided"
 	#getexe \
@@ -197,14 +197,14 @@ getone() {
 	    #sihp1005.img
 	copyright "(c) Copyright Hewlett-Packard 2002"
 	gettgz \
-	    http://foo2zjs.rkkda.com/firmware sihpP1005.tar.gz \
+	    https://www.quirinux.org/printers sihpP1005.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2009"
 	putfw sihp1005.img sihp1005.dl
 	;;
     lj1018|1018)
 	gettgz \
-	    http://foo2zjs.rkkda.com/firmware sihp1018.tar.gz \
+	    https://www.quirinux.org/printers sihp1018.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2005"
 	putfw sihp1018.img sihp1018.dl
@@ -212,28 +212,28 @@ getone() {
     lj1020|1020)
 	# http://ftp.hp.com/pub/softlib/software8/COL14041/lj-39322-2/hp_LJ_1020-1022_Full_Solution.exe
 	gettgz \
-	    http://foo2zjs.rkkda.com/firmware sihp1020.tar.gz \
+	    https://www.quirinux.org/printers sihp1020.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2005"
 	putfw sihp1020.img sihp1020.dl
 	;;
     [pP]100[57])
 	gettgz \
-	    http://foo2zjs.rkkda.com/firmware sihpP1005.tar.gz \
+	    https://www.quirinux.org/printers sihpP1005.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2009"
 	putfw sihpP1005.img sihpP1005.dl
 	;;
     [pP]100[68])
 	gettgz \
-	    http://foo2zjs.rkkda.com/firmware sihpP1006.tar.gz \
+	    https://www.quirinux.org/printers sihpP1006.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2009"
 	putfw sihpP1006.img sihpP1006.dl
 	;;
     [pP]1505)
 	gettgz \
-	    http://foo2zjs.rkkda.com/firmware sihpP1505.tar.gz \
+	    https://www.quirinux.org/printers sihpP1505.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2009"
 	putfw sihpP1505.img sihpP1505.dl
@@ -241,7 +241,7 @@ getone() {
     2200dl|2200)
 	if true; then
 	    gettgz \
-		http://foo2zjs.rkkda.com/icm dl2300.tar.gz \
+		https://www.quirinux.org/printers dl2300.tar.gz \
 		""
 	else
 	    getexe \
@@ -254,7 +254,7 @@ getone() {
     2300dl|2300)
 	if true; then
 	    gettgz \
-		http://foo2zjs.rkkda.com/icm dl2300.tar.gz \
+		https://www.quirinux.org/printers dl2300.tar.gz \
 		""
 	else
 	    getexe \
@@ -262,7 +262,7 @@ getone() {
 		"*.icm"
 	fi
 	gettgz \
-	    http://foo2zjs.rkkda.com/icm km2430.tar.gz \
+	    https://www.quirinux.org/printers km2430.tar.gz \
 	    ""
 	copyright "(c) Copyright Minolta-QMS 2001"
 	puticm foo2zjs
@@ -279,7 +279,7 @@ getone() {
     cpwl|pageworks)
 	if true; then
 	    gettgz \
-		http://foo2zjs.rkkda.com/icm dl2300.tar.gz \
+		https://www.quirinux.org/printers dl2300.tar.gz \
 		""
 	else
 	    getexe \
@@ -298,30 +298,30 @@ getone() {
 	;;
     2430)
 	gettgz \
-	    http://foo2zjs.rkkda.com/icm km2430.tar.gz \
+	    https://www.quirinux.org/printers km2430.tar.gz \
 	    ""
 	copyright "(c) Copyright Konica Minolta 2003"
 	puticm foo2zjs
 	;;
     2530|2490|6115|2480|1600w|1680|1690|4690|110|6121)
 	gettgz \
-	    http://foo2lava.rkkda.com/icm km2530.tar.gz \
+	    https://www.quirinux.org/printers km2530.tar.gz \
 	    ""
 	copyright "(c) Copyright Konica Minolta 2005"
 	copyright "(c) Copyright Rick Richardson 2007.  All Rights Reserved"
 	gettgz \
-	    http://foo2lava.rkkda.com/icm km-1600.tar.gz \
+	    https://www.quirinux.org/printers km-1600.tar.gz \
 	    ""
 	copyright "(c) Copyright Rick Richardson 2009.  All Rights Reserved"
 	puticm foo2lava
 	;;
     1500)
 	gettgz \
-	    http://foo2hp.rkkda.com/icm hpclj2500.tar.gz \
+	    https://www.quirinux.org/printers hpclj2500.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2002"
 	gettgz \
-	    http://foo2hp.rkkda.com/icm hpclj2600n.tar.gz \
+	    https://www.quirinux.org/printers hpclj2600n.tar.gz \
 	    ""
 	copyright "(c) Copyright Hewlett-Packard 2003"
 	copyright "(c) Copyright Rick Richardson 2006.  All Rights Reserved"
@@ -329,33 +329,33 @@ getone() {
 	;;
     1025)
 	gettgz \
-	    http://foo2zjs.rkkda.com/icm hp-cp1025.tar.gz \
+	    https://www.quirinux.org/printer hp-cp1025.tar.gz \
 	    ""
 	copyright "(c) Copyright Rick Richardson 2010.  All Rights Reserved"
 	puticm foo2zjs
 	;;
     1215|1600|2600*)
 	gettgz \
-	    http://foo2hp.rkkda.com/icm hpclj2600n.tar.gz \
+	    https://www.quirinux.org/printer hpclj2600n.tar.gz \
 	    ""
 	gettgz \
-	    http://foo2zjs.rkkda.com/icm km2430.tar.gz \
+	    https://www.quirinux.org/printer km2430.tar.gz \
 	    "km2430_2.icm"
 	copyright "(c) Copyright Hewlett-Packard 2003"
 	copyright "(c) Copyright Rick Richardson 2006.  All Rights Reserved"
 	gettgz \
-	    http://foo2hp.rkkda.com/icm hp1215.tar.gz \
+	    https://www.quirinux.org/printer hp1215.tar.gz \
 	    ""
 	copyright "(c) Copyright Rick Richardson 2008.  All Rights Reserved"
 	puticm foo2hp
 	;;
     300|310|315|325|360|365|2160|3160|3175|3185|6110)
 	gettgz \
-	    http://foo2qpdl.rkkda.com/icm samclp300.tar.gz \
+	    https://www.quirinux.org/printer samclp300.tar.gz \
 	    ""
 	copyright "(c) Copyright Rick Richardson 2007.  All Rights Reserved"
 	gettgz \
-	    http://foo2qpdl.rkkda.com/icm samclp315.tar.gz \
+	    https://www.quirinux.org/printer samclp315.tar.gz \
 	    ""
 	copyright "(c) Copyright Rick Richardson 2008.  All Rights Reserved"
 	puticm foo2qpdl
@@ -365,7 +365,7 @@ getone() {
 
     500)
 	gettgz \
-	    http://foo2slx.rkkda.com/icm lexc500.tar.gz \
+	    https://www.quirinux.org/printer lexc500.tar.gz \
 	    ""
 	copyright "(c) Copyright Lexmark 2006"
 	puticm foo2slx
@@ -388,49 +388,49 @@ getone() {
 	;;
     301)
 	gettgz \
-	    http://foo2hiperc.rkkda.com/icm okic301.tar.gz \
+	    https://www.quirinux.org/printer okic301.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2012"
 	puticm foo2hiperc
 	;;
     c310)
 	gettgz \
-	    http://foo2hiperc.rkkda.com/icm okic310.tar.gz \
+	    https://www.quirinux.org/printer okic310.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2010"
 	puticm foo2hiperc
 	;;
     c511)
 	gettgz \
-	    http://foo2hiperc.rkkda.com/icm okic511.tar.gz \
+	    https://www.quirinux.org/printer okic511.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2012"
 	puticm foo2hiperc
 	;;
     c810)
 	gettgz \
-	    http://foo2hiperc.rkkda.com/icm okic810.tar.gz \
+	    https://www.quirinux.org/printer okic810.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2012"
 	puticm foo2hiperc
 	;;
     3100|3200|5100|5150|5200|5250)
 	gettgz \
-	    http://foo2hiperc.rkkda.com/icm okic3200.tar.gz \
+	    https://www.quirinux.org/printer okic3200.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2005"
 	puticm foo2hiperc
 	;;
     3300|3400|3530)
 	gettgz \
-	    http://foo2hiperc.rkkda.com/icm okic3400.tar.gz \
+	    https://www.quirinux.org/printer okic3400.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2006"
 	puticm foo2hiperc
 	;;
     5500|5600|5800|160)
 	gettgz \
-	    http://foo2hiperc.rkkda.com/icm okic5600.tar.gz \
+	    https://www.quirinux.org/printer okic5600.tar.gz \
 	    ""
 	copyright "(c) Copyright Okidata 2006"
 	puticm foo2hiperc
